dist/websocketClient.d.ts

```diff
@@ -1,7 +1,7 @@
-import * as WebSocket from "ws";
+import ReconnectingWebsocket from "reconnecting-websocket";
 export declare type Listener = (data: IWebsocketData) => void;
 export declare type EventListener = (...args: any[]) => void;
-export declare type MessageListener = (event: IWebsocketMessageEvent) => void;
+export declare type MessageListener = (event: MessageEvent) => void;
```

Comment:
Where does `MessageEvent` come from? Is it a built-in type?

Also, do we need IWebSocketMessageEvent any more? We probably shouldn't remove it since this library is public and it might break others... But maybe we should track this in a github issue?